src/vma/dev/ring_eth_cb.cpp

```diff
@@ -225,6 +225,19 @@ inline bool ring_eth_cb::reload_wq()
 	return false;
 }
 
+int ring_eth_cb::cyclic_buffer_is_readable()
+{
+	volatile struct mlx5_cqe64 *cqe64;
+	uint16_t size = 0;
+	uint32_t flags = 0;
+
+	int ret = ((cq_mgr_mp *)m_p_cq_mgr_rx)->poll_mp_cq(size, m_curr_wqe_used_strides, flags, cqe64);
+	if (unlikely(ret == -1)) {
+		return ret;
+	}
+	return size;
```

Comment:
The vma_cyclic_buffer_is_readable is not needed in VMA and the functionality will be moved to the adaptation layer.